.nvmrc

```diff
@@ -1 +1 @@
-10
+20.12.2
```

Comment:
The latest is 20.16.0. Please check.
Anyway, I've addressed this in my PR.

Reply:
done